sphinx/writers/latex.py

```diff
@@ -365,13 +388,22 @@ def get_colspec(self):
             return '{|' + ('l|' * self.colcount) + '}\n'
 
     def add_cell(self, height, width):
+        """Adds a new cell to a table.
+
+        It will be located at current position: (``self.row``, ``self.col``).
+        """
         self.cell_id += 1
         for col in range(width):
             for row in range(height):
                 assert self.cells[(self.row + row, self.col + col)] == 0
                 self.cells[(self.row + row, self.col + col)] = self.cell_id
```

Comment:
Yes, we have to call this method order by top to bottom and left to right.
The order is same as the order of the doctree: The table node has multiple rows, and each row has multiple columns.